languages/ruby/exercises/concept/exceptions/.docs/instructions.md

```diff
@@ -0,0 +1,33 @@
+In this exercise you will be building error handling for a simple calculator.
+
+The goal is to have a working calculator that returns a string with the following pattern: `16 + 51 = 67`, when provided with arguments `16`, `51` and `+`.
+
+```ruby
+SimpleCalculator.calculate(16, 51, "+"); // => returns "16 + 51 = 67"
+SimpleCalculator.calculate(32, 6, "*"); // => returns "32 * 6 = 192"
+SimpleCalculator.calculate(512, 4, "/"); // => returns "512 / 4 = 128"
+```
+
+## 1. Handle the code that may raise errors within the method `calculate`
+
+The main method for implementation in this task will be the class method `SimpleCalculator.calculate()` method. It takes three arguments. The first two arguments are numbers on which an operation is going to be conducted. The third argument is of type string and for this exercise it is necessary to implement the following operations:
+
+- addition using the `+` string
+- multiplication using the `*` string
+- division using the `/` string
```

Comment:
Can we remove the division operation?  It matches the idea that we don't really have subtraction, we just add negative numbers.  The same is true for division, we multiply fractional numbers.  It keeps the exercise simple, while not removing any functionality.

Reply:
Isn't the division operator actually the key operator here, as later on the `ZeroDivisionError` it raises when trying to divide by 0 is thrown.

Reply:
That is true.  But it isn't necessary for calculations.  But yeah, definitely needs to be here if we a the error must be raised at some point.  (I don't know how to cause a division by 0 error using only multiplication.)